### ISSUE TITLE
Set drawer orientation manually for RTL languages

### ISSUE THREAD

**Issue by bev-on** [eclipsesource/tabris-js#628]
For example if system language in Android is Arabic or Persian, drawer is automatically located right side of app even if app language is English only.

![aaa](https://cloud.githubusercontent.com/assets/11978242/9980672/ea6288d0-5fa9-11e5-80fe-da399214f7a9.png)

How van I fix this issue? If app is designed for English and LTR orientation I want to keep it that way and not be depended on system defaults. If I have Arabic language option and customer choose it I want to move drawer to right side even if system language is English.

Basically what I want is to ignore system language defaults and control drawer orientation manually.

**Comment by mpost** [comment]
Should be doable. We would need to add Tabris.js API for that.

**Comment by bev-on** [comment]
@mpost I guess it must be easy for android. If we have options to set android:layoutDirection and android:textDirection it will be great.
http://developer.android.com/reference/android/view/View.html#attr_android:layoutDirection
http://developer.android.com/reference/android/view/View.html#attr_android:textDirection

I'm sure it can be done for iOS as well.
https://developer.apple.com/library/ios/documentation/MacOSX/Conceptual/BPInternational/SupportingRight-To-LeftLanguages/SupportingRight-To-LeftLanguages.html
https://developer.apple.com/library/ios/documentation/MacOSX/Conceptual/BPInternational/InternationalizingYourUserInterface/InternationalizingYourUserInterface.html#//apple_ref/doc/uid/10000171i-CH3-SW3

**Comment by mpost** [comment]
We are currently looking into that issue again... Did you have any other problems/feature requests in regards on to right to left support?

**Comment by bev-on** [comment]
@mpost for now nothing else. We need to set page orientation but If we will able to set orientation of different widgets manually it will be great too... Sometimes even if all application is designed for RTL language, there can be widget with English text as well (list of countries for example in collection view). So if page orientation is set to RTL and if we still will be able to change orientation of some widgets to LTR it will be great. There is already "alignment" property for texts and if there is possibility please add "direction" or "orientation" property as well.

**Comment by mpost** [comment]
@bev-on thansk for your input.

Setting the reading direction on a per widget basis also makes sense as you have described on your comment. I have opened an issue for that: https://github.com/eclipsesource/tabris-js/issues/654

We are considering to make improve right to left support by adding complementary layout properties like start and end so that layouting can be more dynamic. New issue: https://github.com/eclipsesource/tabris-js/issues/655

**Comment by bev-on** [comment]
@mpost any updates on this feature?

**Comment by mpost** [comment]
@bev-on No news atm.

**Comment by StetHD** [comment]
@mpost Does Tabris support this feature now?

**Comment by mpost** [comment]
@StetHD Currently the drawer is still left oriented.
